site/en/integrations/integrate_with_sentencetransformers.md

```diff
@@ -46,34 +46,30 @@ At this point, we are going to begin setting up Milvus. The steps are as follows
 1. Create a Milvus Lite database in a local file. (Replace this URI to the server address for Milvus Standalone and Milvus Distributed.)
 
 ```python
-connections.connect(uri="./sentence_transformers_example.db")
+client = MilvusClient(uri="./sentence_transformers_example.db")
 ```
 
 2. Create the data schema. This specifies the fields that comprise an element including the dimension of the vector embedding.
 
 ```python
 fields = [
-    FieldSchema(name='id', dtype=DataType.INT64, is_primary=True, auto_id=True),
-    FieldSchema(name='title', dtype=DataType.VARCHAR, max_length=256),
-    FieldSchema(name='embedding', dtype=DataType.FLOAT_VECTOR, dim=embedding_dim)
+    FieldSchema(name="id", dtype=DataType.INT64, is_primary=True, auto_id=True),
+    FieldSchema(name="title", dtype=DataType.VARCHAR, max_length=256),
+    FieldSchema(name="embedding", dtype=DataType.FLOAT_VECTOR, dim=embedding_dim),
+    FieldSchema(name="year", dtype=DataType.INT64),
+    FieldSchema(name="origin", dtype=DataType.VARCHAR, max_length=64),
 ]
 
 schema = CollectionSchema(fields=fields, enable_dynamic_field=False)
-collection = Collection(name=collection_name, schema=schema)
+client.create_collection(collection_name=collection_name, schema=schema)
 ```
 
-3. Define the vector search indexing algorithm. Milvus Lite implements brute force search and HNSW, whereas Milvus Standalone and Milvus Distributed implement a wide variety of methods. For this scale of data, the naive brute force search suffices.
+3. Define the vector search indexing algorithm. Milvus Lite support FLAT index type, whereas Milvus Standalone and Milvus Distributed implement a wide variety of methods such as IVF, HNSW and DiskANN. For the small scale of data in this demo, any search index type suffices so we use the simplest one FLAT here.
 
 ```python
-params = {
-    'index_type':"FLAT",
-    'metric_type': "IP"
-    }
-
-collection.create_index(
-    'embedding',
-    params
-)
+index_params = client.prepare_index_params()
```

Comment:
let's add scalar index for fields used for metadata filter and explain that it will make the search faster https://milvus.io/docs/scalar_index.md. (technically for milvus lite which uses brute search on vector the scalar index doesn't make much difference but i'd like to educate that as a good habit for production, otherwise it's gonna be misleading for docker/k8s users)